backend/src/test/java/de/jmpsoftware/backend/model/PipeEntryTest.java

```diff
@@ -0,0 +1,112 @@
+package de.jmpsoftware.backend.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PipeEntryTest {
+
+    @Test
+    void PackedInteger()
+    {
+        new PipeEntry();
+        PipeEntry pipeEntry = PipeEntry.builder()
+                .command(0)
+                .valueDMX(255)
+                .channel(0)
+                .universe(1)
+                .build();
+        new PipeEntry();
+        PipeEntry newPipeEntry = PipeEntry.builder()
+                .command(0)
+                .valueDMX(0)
+                .channel(0)
+                .universe(0)
+                .build();
+
+        int packedInteger = pipeEntry.getPackedIntegerByValues();
+        newPipeEntry.setValuesByPackedInteger(packedInteger);
+
+        assertEquals(pipeEntry, newPipeEntry);
+
+        new PipeEntry();
+        pipeEntry = PipeEntry.builder().command(2).valueDMX(90).channel(23).universe(1).build();
+        new PipeEntry();
+        newPipeEntry = PipeEntry.builder().command(0).valueDMX(0).channel(0).universe(0).build();
+        packedInteger = pipeEntry.getPackedIntegerByValues();
+        newPipeEntry.setValuesByPackedInteger(packedInteger);
+        assertEquals(pipeEntry, newPipeEntry);
+
+
+        new PipeEntry();
+        pipeEntry = PipeEntry.builder().command(111).valueDMX(0).channel(0).universe(16).build();
+        new PipeEntry();
+        newPipeEntry = PipeEntry.builder().command(0).valueDMX(0).channel(0).universe(0).build();
+        packedInteger = pipeEntry.getPackedIntegerByValues();
+        newPipeEntry.setValuesByPackedInteger(packedInteger);
+        assertNotEquals(pipeEntry, newPipeEntry);
```

Comment:
Maybe research parameterized tests to get rid of some repetition? https://www.baeldung.com/parameterized-tests-junit-5